### PR TITLE
Polish language

### PR DESCRIPTION
**Pull request by aPoCoMiLogin**

### CODE REVIEW

File: src/locales/pl.json [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+      "quit": "Zamknij"
+    },
+    "edit": {
+      "title": "Edycja",
+      "undo": "Cofnij",
+      "redo": "Ponów",
+      "paste": "Wklej",
+      "cut": "Wytnij",
+      "copy": "Kopiuj",
+      "copy_link": "Kopiuj adres linku",
```
Comment (averissimo) [inline src/locales/pl.json]: missing "copy_image_link": "Copy Image Link",

Comment (averissimo) [inline src/locales/pl.json]: above `copy_link`

File: src/locales/pl.json [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+      "copy_image": "Kopiuj grafikę",
+      "select_all": "Zaznacz wszystko",
+      "language": "Język"
```
Comment (averissimo) [inline src/locales/pl.json]: can you change to: I'll have to change the template as this is not obvious and I've been doing this personally

"language": "Język (Language)"

File: src/locales/pl.json (hunk 1 above)
Comment (aPoCoMiLogin) [inline src/locales/pl.json]: oops, had old `en.json` file as base

File: src/locales/pl.json (hunk 2 above)
Comment (aPoCoMiLogin) [inline src/locales/pl.json]: sure

**Pull request merged**